statsmodels/graphics/gofplots.py

```diff
@@ -178,20 +187,25 @@ def sorted_data(self):
         return sorted_data
 
     @cache_readonly
-    def sample_quantiles(self):
-        if self.fit and self.loc != 0 and self.scale != 1:
-            return (self.sorted_data-self.loc)/self.scale
+    def scaled_data(self):
+        if self.loc == 0 and self.scale == 1:
+            return (self.sorted_data-np.mean(self.data))/np.std(self.data)
         else:
-            return self.sorted_data
+            return self.sample_quantiles
+
+    @cache_readonly
+    def sample_quantiles(self):
+        return self.sorted_data #(self.sorted_data-self.loc)/self.scale
 
     @cache_readonly
     def sample_percentiles(self):
-        quantiles = \
-            (self.sorted_data - self.fit_params[-2])/self.fit_params[-1]
-        return self.dist.cdf(quantiles)
+        if self.fit or self._userdist_is_frozen:
+            return self.dist.cdf(self.sorted_data)
+        else:
+            return self.dist.cdf(self.scaled_data)
```

Comment:
Note, the backwards incompatible change here is to replace standardization by fit_params (which might be 0,1), by the scaled_data which is scaled/standardized base on sample mean and variance. (a few lines higher up)